src/classes/BDI_MatchDonations.cls

```diff
@@ -468,30 +468,32 @@ public with sharing class BDI_MatchDonations implements BDI_IMatchDonations {
 
         // now query for all Open Opportunities from these Contacts or Accounts
         integer cFilter = 0;
-        string strSoql = 'select ' + string.join(new list<String>(setOppFields), ',');
+        string strSoql = 'SELECT ' + string.join(new list<String>(setOppFields), ',');
         
         // add Payment subquery
-        strSoql += ', (select ' + string.join(new list<String>(setPmtFields), ',');
-        strSoql += ' from npe01__OppPayment__r where npe01__Paid__c = false )';
-        
-        strSoql += ' from Opportunity';
-        strSoql += ' where IsClosed = false and (';
+        strSoql += ', (SELECT ' + string.join(new list<String>(setPmtFields), ',');
+        strSoql += ' FROM npe01__OppPayment__r ' +
+                'WHERE npe01__Paid__c = false ' +
+                'AND npe01__Written_Off__c = false)'; //Do not include written-off Payments.
```

Comment:
This line (AND npe01__Written_Off__c = false) is the only change not related to formatting and style, right?